SPECS/skopeo/skopeo.spec

```diff
@@ -46,6 +46,9 @@ make test-unit-local
 %{_mandir}/man1/%%{name}*
 
 %changelog
+* Tue Feb 13 2024 David Steele <davidsteele@microsoft.com> - 1.14.2-1
```

Comment:
Test build: [PR-7859](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=504823&view=results)

Reply:
@mfrw that pipeline has passed. Do you require anything more from me to get this PR merged?

Reply:
Let's wait for a review from one of the stable maintainers.
/cc @jslobodzian @christopherco

Reply:
Bump on this @jslobodzian/ @christopherco. Could you please review?

Reply:
Is is this MR ready to go  @jslobodzian / @christopherco ?